db/crud.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy.orm import Session
+from db.models import DBAuthor, DBBook
+from db.schemas import AuthorCreate, BookCreate
+
+
+def get_all_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(DBAuthor).offset(skip).limit(limit).all()
+
+
+def get_author(db: Session, author_id: int):
+    return db.query(DBAuthor).filter(DBAuthor.id == author_id).first()
```

Comment:
Consider adding error handling for cases where the author with the given `author_id` does not exist. This will help in providing more informative feedback to the caller.